src/services/incomesExpenses.ts

```diff
@@ -28,3 +28,25 @@ export const createIncomeExpense = async (reqBody: CreateIncomeExpense, isExpens
     throw new Error(error.message);
   }
 };
+
+export const deleteIncomeExpense = async (incomeExpenseId: string, isExpense: boolean) => {
```

Comment:
I think if you create a service for that, it will be easier to maintain and you will write much less code. In the service you can pass just the route and create 3 methods (GET, POST, DELETE), adding default properties inside service like error and headers...